src/coreclr/debug/createdump/configure.cmake

```diff
@@ -1,3 +1,22 @@
 check_function_exists(process_vm_readv HAVE_PROCESS_VM_READV)
 
+check_symbol_exists(
```

Comment:
This feedback doesn't need to block this check-in by any means. We should probably start sharing the checks in the native portion of the engineering folder. Seems silly to have 4-5 copies of the same check

Reply:
`src\native\minipal` is meant for things like this.

Reply:
IIRC, @janvoli prefers to keep these checks separated per-project. I think what's non-controversial is deleting the dead checks and inefficient checks; we have plenty of them. e.g. checking for symbols, headers etc. should not use (the slow) compile/run helpers.